spec/requests/families_spec.rb

```diff
@@ -24,6 +24,42 @@
     end
   end
 
+  describe "GET #show" do
+    let(:family) { create(:family, partner: partner) }
+
+    it "should return status code 200 if a family is found" do
+      get family_path(family.id)
+
+      expect(response).to have_http_status :ok
+    end
+
+    it "should render 404 for a family not found" do
+      max_id = family.id + 1
+
+      get family_path(max_id)
+
+      expect(response).to have_http_status :not_found
+    end
+  end
+
+  describe "GET #edit" do
+    let(:family) { create(:family, partner: partner) }
+
+    it "should render the edit template" do
+      get edit_family_path(family.id)
+
+      expect(response).to render_template(:edit)
```

Comment:
I think that the rails core team probably removed the `assert_template` matcher for a good reason. Id rather not bring these back with the `rails-controller-testing` is there another assertion we could make here that would be just as effective? Perhaps testing that the controller responds with a 200 would be enough?